### PR TITLE
Cleanup Add Event Form

### PR DESCRIPTION
**Pull request by pdw207**
## Add Event Form Submission 
form

### Changes
- Create Thank You Page
- Fix Form Validation which trigger on blur
- Make form a modal per designs with full width on mobile
- Added test coverage
- Pulled post request to prop (if we want to switch it out in the future)

### Mobile Styling
screen shot 2017-04-15 at 10 55 21 pm

### CODE REVIEW

File: src/components/ThankYou/ThankYou.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import styles from './ThankYou.sass';
+
+const ThankYou = () => {
+  return (
+    <div className={styles.thankYouWrapper}>
+      <h1> Thank you! Your submission has been submitted for review. </h1>
```
Comment (aaghevli) [inline src/components/ThankYou/ThankYou.js]: Looks good! Maybe the following?

> Thank you! Your event has been submitted for review.

File: src/components/AddEvent/AddEvent.test.js [hunk 2]
```diff
@@ -1,17 +1,124 @@
...
+          submitDisabled: false
+        });
+      });
+    });
+    describe('email validation', () => {
+      beforeEach(() => {
+        wrapper.setState({ eventLink: 'facebook.com/test', eventName: 'adfsjlk' });
+      });
+
+      it('valid email show no validation warning and enables submit', () => {
```
Comment (dorono) [inline src/components/AddEvent/AddEvent.test.js]: "show'" should be "shows"

File: src/components/ThankYou/ThankYou.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      <Link to="/"> View some our event listing. </Link>
```
Comment (dorono) [inline src/components/ThankYou/ThankYou.js]: Should be "View some of our event listings", or, IMO, "View our event listings".

**Comment by dorono** [comment]
Looks really awesome! Learned some new es6 syntax as well as a bit about testing promises with sinon. :) Aside from a couple of typos in user-facing and test text, I pulled down your branch and noticed a couple of issues with the CSS:
* at mobile portrait view, I think it's happening up 0 to the "sm" breakpoint width, the "ADD EVENT" button moves over to the left, and is no longer in alignment with the popup (see screenshot below)
* also at mobile portrait view, the add event popup seems goes into relative positioning, and pushes down the content, instead of sitting on top of it using absolute or fixed positioning. Not sure if this is by design or not, as I couldn't really tell from looking at the designs on Trello which don't show anything below that popup on modal, but my sense is that it should sit on top and not affect the content flow of the rest of the page.

**Comment by dorono** [comment]
Looks really awesome! Learned some new es6 syntax as well as a bit about testing promises with sinon. :) Aside from a couple of typos in user-facing and test text, I pulled down your branch and noticed a couple of issues with the CSS:
* at mobile portrait view, I think it's happening up 0 to the "sm" breakpoint width, the "ADD EVENT" button moves over to the left, and is no longer in alignment with the popup (see screenshot below)
* also at mobile portrait view, the add event popup seems goes into relative positioning, and pushes down the content, instead of sitting on top of it using absolute or fixed positioning. Not sure if this is by design or not, as I couldn't really tell from looking at the designs on Trello which don't show anything below that popup on modal, but my sense is that it should sit on top and not affect the content flow of the rest of the page.

**Comment by pdw207** [comment]
All comments addressed.

**Pull request merged**